includes/class-wc-payments.php

```diff
@@ -1484,6 +1487,20 @@ function ( $container ) {
 		}
 	}
 
+	/**
+	 * Initializes woopay direct checkout if the woopay feature flag is enabled.
+	 *
+	 * @return void
+	 */
+	public static function maybe_enable_woopay_direct_checkout() {
+		if ( ! WC_Payments_Features::is_woopay_enabled() ) {
+			return;
+		}
+
+		$woopay_direct_checkout = new WC_Payments_WooPay_Direct_Checkout();
+		$woopay_direct_checkout->init();
```

Comment:
Once the feature is enabled, this ends up adding about 2kB to every page load by including the direct checkout script. Given the mini cart needing to be captured, I'm not sure we can escape this, but wanted to mention it in case there is a good solution as script bloat has been a recent complaint in WC and WooPayments.

Reply:
Good to note this, but currently there isn't much of an option otherwise. One thing to note is that once we combine the first party auth and direct to woopay connect iframes we might actually come out even as we'll remove some duplicate iframe code.